src/traces/histogram/attributes.js

```diff
@@ -82,15 +82,21 @@ module.exports = {
         role: 'style',
         description: [
             'Determines whether or not the x axis bin attributes are picked',
-            'by an algorithm.'
+            'by an algorithm. Note that this should be set to false if you',
```

Comment:
Similar to my comment in https://github.com/plotly/plotly.js/pull/607, `autobinx` does not **need** to be set to `false` as it is defaulted to `false` in if `xbins.start` and `xbins.end` are valid inputs (see [logic](https://github.com/plotly/plotly.js/blob/bcde0484c32a6ddc1423f47b044ccc6faa425331/src/traces/histogram/bin_defaults.js#L21)). 

But we don't really have a way to communicate _smart default_ logic in our descriptions yet. 

So, I think this patch is fine for now. But we should starting thinking about a better approach cc @cldougl